utils/install_v2.sh

```diff
@@ -12,6 +12,7 @@ YELLOW=$'\e[0;33m'
 NC=$'\e[0m' # No Color
 TMP_DIR="/tmp/wasmedge.$$"
 
+
```

Comment:
What's the reason to add an empty line here?

Reply:
no specific reason, i made a new PR